src/main/java/uk/gov/hmcts/darts/common/client/AudiosClient.java

```diff
@@ -56,4 +62,20 @@ public ResponseEntity<Void> addAudio(MultipartFile file, AddAudioMetadataRequest
         streamAudio(file, metadata);
         return new ResponseEntity<>(HttpStatusCode.valueOf(200));
     }
+
+    @Override
+    @SuppressWarnings("PMD.LooseCoupling")
+    public ResponseEntity<Void> addAudioMetaData(AddAudioMetadataRequestWithStorageGUID addAudioMetadataRequestWithStorageGuid) {
+        try {
+            HttpHeaders headers = new HttpHeaders();
+            headers.setContentType(MediaType.APPLICATION_JSON);
+            headers.setAccept(List.of());
```

Comment:
Do we need to do this?

Reply:
Yes :)

If we don't reset the accept flag, it will be given application/json by default which will be rejected by the darts-api 
 as this API does not have a response body as such it fails to find an API that can return application/json